programmers/string-handling-basic.js

```diff
@@ -0,0 +1,20 @@
+function solution(s){
+    const linerMeasure = s.length;
+    if(linerMeasure === 4 | linerMeasure === 6){
+        for (let i = 0 ; i<linerMeasure; i++){
+            let store = 0;
+            for (let j = 0 ;  j<10 ; j++){
+                if (s[i] == j){
+                    store = 1;
+                }
+            }
+            if (store === 0){
+                return console.log("false");
+            }
+        }
+        return console.log("true");
+    }
+    console.log("문자열의 길이 조건에 어긋난다.");
+}
+let s ="1234";
+solution(s);
```

Comment:
이 방식으로 푸시려고 하신거 같아서 가희님 처럼 풀었을 때의 정답이 나오려면
밑의 방식처럼 푸셔야합니다.
```suggestion
function solution(s) {
    let answer = false;
    if (s.length === 4 || s.length === 6) { //문자열 길이 4 또는 6 이고
         for (let i = 0; i < s.length; i++){ 
            if (Number.isInteger(parseInt(s[i]))){ //숫자로만 구성
                answer=  true;
            } else {
                answer= false;
                break;
            }
        }
    } else {
        answer = answer; 
    }
    return answer;
}
```